### PR TITLE
Add initial crates

### PR DESCRIPTION
**Pull request by dhardy**
See https://github.com/rust-random/rand/issues/885

Note: this updates CI and repository URLs. It does not update documentation URLs pointing at https://rust-random.github.io/rand/ since we have yet to decide how to handle this: #2

### CODE REVIEW

**Comment by dhardy** [comment]
CI: I simplified the Travis jobs slightly, but potentially we could trim yet more jobs and drop AppVeyor altogether for this repo. Thoughts?

**Comment by vks** [comment]
I would like to keep AppVeyor around for `rand_jitter`, because timers work differently there.

File: benches/mod.rs [hunk 1]
```diff
@@ -54,6 +54,24 @@ gen_bytes!(gen_bytes_splitmix64, SplitMix64::from_entropy());
 gen_bytes!(gen_bytes_isaac, IsaacRng::from_entropy());
 gen_bytes!(gen_bytes_isaac64, Isaac64Rng::from_entropy());
 
+// Save a dependency on Rand:
+trait Generable {
```
Comment (vks) [inline benches/mod.rs]: I think `Generate` would be a more idiomatic name.

**Comment by dhardy** [comment]
AppVeyor [configured](https://ci.appveyor.com/project/rust-random/rngs). Travis appears to need time for synchronisation.

**Pull request merged**